frontend/src/components/admin-page/users.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import { AppRoute } from 'common/enums';
+import { RootState } from 'common/types';
+import { UsersActionCreator } from 'store/slices';
+import { Column } from 'common/interfaces';
+import Table from './table';
+
+
+const checkIdentifierType = (identifier:string):Column =>{
+    return {
+      Header: identifier.charAt(0).toUpperCase() + identifier.slice(1),
```

Comment:
You can do it with styles 
https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform